src/fparser/common/tests/test_sourceinfo.py

```diff
@@ -51,17 +51,16 @@
 
 ##############################################################################
 
-# pylint: disable=unused-variable
 def test_format_constructor_faults():
     '''
     Tests that the constructor correctly rejects attempts to create an object
     with "None" arguments.
     '''
     with pytest.raises(Exception):
-        unit_under_test = FortranFormat(True, None)
+        _unit_under_test = FortranFormat(True, None)
```

Comment:
I meant just assign to "_" rather than "_unit_under_test" but if the latter is sufficient to keep pylint happy then I'm happy.